lib/functions/rloiProcess.js

```diff
@@ -15,110 +10,9 @@ module.exports.handler = async (event) => {
     const key = event.Records[0].s3.object.key
```

Comment:
There's a bunch of these deep property accessing

E.g. `event.Records[0].s3.object.key`

Obviously this will throw an unhandled exception if any part of the path is undefined, or Records isn't a array.
I'll not point them all out and if you trust the data source then no worries, otherwise add some guards/defensive coding.

Reply:
That's by design, if the event doesn't have the correct parameters coming into lambda function I want it to throw